### PR TITLE
update CHANGELOG

### PR DESCRIPTION
**Pull request by japaric**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,8 +7,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Added
+
+- [#662]: `#[derive(Format)]` now accepts attribute fields to format fields using the `Debug2Format` adapter instead of a `Format` implementation.
+- [#656] implement `Format` for `Cell` and `RefCell`
+- [#661] add tests for Cell types
```
Comment (jonathanpallant) [inline CHANGELOG.md:14]: Is there a link for #661?

**Review by Urhengulas — APPROVED** [review]
What @jonathanpallant said, but else looks good!

**Comment by japaric** [comment]
added missing link :chains:

**Review by Dajamante — APPROVED** [review]

**Comment by japaric** [comment]
bors retry

**Pull request merged**